docs/pages/installation.mdx

```diff
@@ -48,56 +49,345 @@ supports most features on Windows 10 and later.*
+   $ sudo yum install ${TELEPORT_PKG?}
+   #
+   # Tip: Add /usr/local/bin to path used by sudo (so 'sudo tctl users add' will work as per the docs)
+   # echo "Defaults    secure_path = /sbin:/bin:/usr/sbin:/usr/bin:/usr/local/bin" > /etc/sudoers.d/secure_path
+   ```
+   
+   </TabItem>
+   
+   <TabItem label="zypper">
+   
+   ```code
+   # Add the Teleport Zypper repository. You'll need to update this file for each
+   # major release of Teleport.
+   # First, get the OS major version from $VERSION_ID so this fetches the correct
+   # package version.
+   $ VERSION_ID=$(echo $VERSION_ID | grep -Eo "^[0-9]+")
+   # Use zypper to add the teleport RPM repo
+   $ sudo zypper addrepo --refresh --repo $(rpm --eval "https://zypper.releases.teleport.dev/$ID/$VERSION_ID/Teleport/%{_arch}/stable/${TELEPORT_VERSION?}/teleport-zypper.repo")
+   $ sudo yum install ${TELEPORT_PKG?}
+   #
+   # Tip: Add /usr/local/bin to path used by sudo (so 'sudo tctl users add' will work as per the docs)
+   # echo "Defaults    secure_path = /sbin:/bin:/usr/sbin:/usr/bin:/usr/local/bin" > /etc/sudoers.d/secure_path
+   ```
+   
+   </TabItem>
+   
+   <TabItem label="dnf">
+   
+   ```code
+   # Add the Teleport YUM repository for v(=teleport.major_version=). You'll need to update this
+   # file for each major release of Teleport.
+   # First, get the major version from $VERSION_ID so this fetches the correct
+   # package version.
+   $ VERSION_ID=$(echo $VERSION_ID | grep -Eo "^[0-9]+")
+   # Install dnf config-manager
+   $ sudo yum install -y yum-utils
+   # Use the dnf config manager plugin to add the teleport RPM repo
+   $ sudo dnf config-manager --add-repo "$(rpm --eval "https://yum.releases.teleport.dev/$ID/$VERSION_ID/Teleport/%{_arch}/stable/${TELEPORT_VERSION?}/teleport.repo")"
+   
+   # Install teleport
+   $ sudo dnf install ${TELEPORT_PKG}
+   
+   # Tip: Add /usr/local/bin to path used by sudo (so 'sudo tctl users add' will work as per the docs)
+   # echo "Defaults    secure_path = /sbin:/bin:/usr/sbin:/usr/bin:/usr/local/bin" > /etc/sudoers.d/secure_path
+   ```
+   
+   </TabItem>
+   </Tabs>
+
+### TAR archives
+
+For self-hosted deployments, Teleport maintains TAR archives for
+Linux-compatible binaries at `https://cdn.teleport.dev`. This section explains
+the Teleport TAR archives and how to use them.
+
+<Notice type="warning">
+
+It is not possible to install the automatic agent updater using TAR archives.
+Teleport Cloud customers must use the [one-line installation
+script](#one-line-installation-script) or manually install Teleport from a
+[package repository](#package-repositories) in order to install the updater.
+
+</Notice>
+
+1. Assign environment variables that you will use to download your intended
+   archive:
+
+   <Tabs>
+   <TabItem label="Teleport Community Edition">
```

Comment:
When reading through this I actually missed the "warning" above and was looking for a "Teleport Cloud" tab. What do you think about adding the "Teleport Cloud" tab here as well and adding the same warning text into it saying that Cloud users need to install using script or a package? I think it would draw their attention better than just a warning since they would (hopefully) click on the Cloud tab.

Reply:
I found this a little tricky because the `Tabs` component only applies to assigning environment variables, not the whole procedure. I've removed the `Tabs` component and suggested a way to do this through regular body text so readers aren't tempted to skip the intro text.